_posts/en/newsletters/2019-03-19-newsletter.md

```diff
@@ -0,0 +1,277 @@
+
+    Overall, it's great to see developers building tools that help
+    people identify flaws in their software or their behavior, but it's
+    also important to consider how users will interact with the tool.
+    As Wuille said near the end of the conversation: "I am super happy
+    that there is a decent explorer now for debugging stuff out there,
+    but I'm concerned about making it sound like it's an actual
+    production tool.  I know people will use explorers, and one that
+    gives good information is better than one that confuses everything.
+    But, really, we shouldn't encourage using [it].  If this privacy
+    detection feature causes people to go look up all their transactions
+    because of a gamification like feeling 'oooh let's see how my
+    transaction did here?!', it's probably a net negative. [...] The
+    most important thing to put on a block explorer is 'Warning: looking
+    up your own addresses on a block explorer leaks your privacy to the
+    site operator'."
+
+- **Spy node ban list updated:** some IP addresses have opened an
+  excessive number of connections to other nodes, probably as part of
+  attempts to monitor transaction propagation to determine which nodes
+  originated which transactions.  Other IP addresses run software that
+  signals poor intentions in other ways.  To help node operators refuse
+  connections from those IP addresses, Gregory Maxwell maintains a
+  ban list that can be imported into Bitcoin Core and compatible nodes.
+  There is absolutely no need to use this centralized list---your fully
+  decentralized node will attempt to connect to a diverse enough set of
+  peers that it should establish at least one honest connection---but
+  using this ban list may reduce the amount of traffic you waste on spy
+  nodes and other bad actors.  The list comes in two formats, one for
+  use on the command line with [bitcoin-cli][banlist cli] and one that
+  can be pasted into the debug console of [Bitcoin Core GUI][banlist
+  gui].  The blacklisted IP addresses are banned for one year and
+  Bitcoin Core will remember the bans between restarts, so you only need
+  to import the list once.
+
+- **MIT Bitcoin Club 2019 Expo videos available:** a series of talks
+  from the exposition two weeks ago have been split into individual
+  [videos][mit vids] and uploaded to YouTube.  We've heard that many of
+  the talks were excellent, so consider browsing the playlist for topics
+  that sound interesting to you.
+
+## Bech32 sending support
+
+*Week 1 of 24*
+
+Bech32 native segwit addresses were first publicly [proposed][bech32
+proposed] almost exactly two years ago, becoming the [BIP173][]
+standard.  This was followed by the segwit soft fork's lock-in on 24
+August 2017.  Yet, seventeen months after lock-in, some popular wallets
+and services still don't support sending bitcoins to bech32 addresses.
+Developers of other wallets and services are tired of waiting and want
```

Comment:
An alterative expression, arguably more positive, would be "Developers of other wallets and services are anxiously waiting"